wagtail/admin/templatetags/trans_filter.py

```diff
@@ -0,0 +1,10 @@
+from django import template
+from django.utils.translation import gettext as _
+
+register = template.Library()
+
+
+@register.simple_tag
+def trans(msg_var):
+    result_var = msg_var
+    return _(result_var)
```

Comment:
Hey. This might work for known translations but I'm not sure this will work for new translations. 


See how make translations works. https://docs.djangoproject.com/en/4.1/ref/django-admin/#django-admin-makemessages

See also https://docs.djangoproject.com/en/4.1/topics/i18n/translation/

Reply:
I might suggest starting with a few unit tests that also check whether the make translations script will parse them.